runtime/src/test/java/jdocs/akka/grpc/client/GrpcClientSettingsCompileOnly.java

```diff
@@ -23,12 +26,13 @@ public static void sd() {
                 .withTls(false);
         //#simple-programmatic
 
+        SimpleServiceDiscovery serviceDiscovery= ServiceDiscovery$.MODULE$.apply(actorSystem).discovery();
+
```

Comment:
This is not really necessary since it's not printed in docs and could be simply `SimpleServiceDiscovery serviceDiscovery= null;`

Reply:
This could be `ServiceDiscovery.get(as).discovery()`, right? In that form it might be acceptable to include in the example.

Reply:
Or pass in for example `ConfigSimpleServiceDiscovery(system)`

Reply:
Sure! Fixing.

I prefer not passing `ConfigSimpleServiceDiscovery` since it's a specific implementation and use what's loaded by the extension instead.